examples/example.yml

```
@@ -47,3 +47,6 @@
     - name: always run
       debug: msg="always_run is deprecated"
       always_run: true
+
+    # empty task is currently accepted by ansible as valid code:
+    -
```

```suggestion
    - name: empty
```
This test is also covered, correct me otherwise.

---

Putting a name there would render the test useless because it will produce a dictionary with a key name in it. We need a real `null` there for testing, not even an empty dictionary would test it correctly.